build.js

```diff
@@ -11,9 +11,23 @@ const { zlib } = require("mz");
  */
 process.on("SIGQUIT", () => process.exit(0));
 
+const [opts, args] = cliopts.parse(
+  ["production", "Run in production mode"],
+  ["settings", "A JSON config file for the server", "<file>"],
+  ["mongoUrl", "A mongoDB connection connection string", "<url>"],
+  ["mongoUrlFile", "The name of a text file which contains a mongoDB URL for the database", "<file>"],
+  ["postgresUrl", "A postgresql connection connection string", "<url>"],
+  ["postgresUrlFile", "The name of a text file which contains a postgresql URL for the database", "<file>"],
+  ["shell", "Open an interactive shell instead of running a webserver"],
+  ["command", "Run the given server shell command, then exit", "<string>"],
+);
+
 const defaultServerPort = 3000;
 
 const getServerPort = () => {
+  if (opts.command) {
+    return 5001;
+  }
```

Comment:
Just a random number that's less likely to be in use than 3000 or 4000 so that we don't get an error if we try to run migrations whilst a dev server is running.